litani

```diff
@@ -537,13 +545,23 @@ def init(args):
     except KeyError:
         run_id = str(uuid.uuid4())
 
-    cache_dir = pathlib.Path(
-        tempfile.gettempdir()) / "litani" / "runs" / run_id
+    if args.output_directory:
+        output_directory = pathlib.Path(args.output_directory).resolve()
+    else:
+        output_directory = pathlib.Path(tempfile.gettempdir())
+
+    cache_dir = output_directory / "litani" / "runs" / run_id
```

Comment:
I actually had in mind that `cache_dir` would be the path that the user provided, not a subdirectory 3 levels beneath it.

However, there is an advantage to doing it your way: users would not have to specify a fresh directory every time. I was assuming that users would do something like
```
litani init --output-directory output/$(date +%Y-%m-%dT%H:%M:%S)
```
and then the files would be written directly to that directory. Both are valid approaches, I suppose.

Here's what I suggest.
* Make `--output-directory` do what I'm suggesting.
* If you think it's worth it, add a third flag `--output-prefix` which does what you're doing up there. I do think it's nice to have, although I'm slightly wary of having too many flags, and I think what I proposed does give users more control.
* The two flags should be mutually exclusive.